src/dev-mock-service/README.md

```diff
@@ -0,0 +1,47 @@
+## Dev Mock Service
+
+Dev Mock Service Worker is an API mocking library that allows you to specify
+custom responses for any API calls inside your app.
+
+### Capabilities
+
+- Mock API requests
+- Dev/Prod switch
+
+### Installation
+
+Add the line `import { DevMockMode, DevMock } from '@devvit/kit';` in the beginning of your root component.
+
+### API mocks
+
+With custom handler functions, you can override the response for any API call, such as
+Redis, RedditAPI, or HTTP request.
+
+- In Dev mode (`DevMockMode.Dev`), handlers are applied for all requests with the matching method and ID (if available).
+- In Prod mode (`DevMockMode.Prod`), handlers are ignored.
+
+#### Setup
+
+Create devv versions of the API clients you want to mock.
```

Comment:
Is this in the test file or in the real app code?

Reply:
Real app code, this molecule is not intended to be used in unit tests (however it certainly can be)